server/util/ioutil/ioutil.go

```diff
@@ -71,22 +73,20 @@ func (c *CustomCommitWriteCloser) Commit() error {
 }
 
 func (c *CustomCommitWriteCloser) Close() error {
-	var firstErr error
```

Comment:
Why can't we continue to use this as a local variable?

Reply:
I think I could have misunderstood the implementation here.

Let me fix it